packages/r/reseter.css.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "reseter.css",
+  "description": "Reset All CSS By Browsers With Reseter.css. And Make Web Look Same Across All Browsers",
+  "keywords": [
+    "Reset.css",
+    "Normalize.css",
+    "Sanitize.css",
+    "Reboot"
+  ],
+  "homepage": "https://krishdevdb.github.io/reseter.css/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/krishdevdb/reseter.css.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "reseter.css",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/reseter.@(css|min.css)",          
+          "src/**/reseter.@(sass|scss|less|styl|ts|js|mjs)" 
+        ]
+      }
+    ]
+  },
+  "filename": "css/reseter.min.css"
```

```suggestion
  "filename": "reseter.min.css"
```
Will be needed if hoisted out